testdata/lineage/basic-multiversion.txtar

```diff
@@ -1840,3 +1840,59 @@ type Basicmultiversion struct {
 
 // BasicmultiversionWithDefault defines model for Basicmultiversion.WithDefault.
 type BasicmultiversionWithDefault string
+
+-- out/encoding/typescript/TestGenerate/nilcfg --
+export interface Basic-Multiversion {
```

Comment:
I don't think a Typescript identifier can contain `-`

Reply:
We didn't add anything in Cuetsy to normalise the fields and it should do it.

Reply:
So, do you want me to create an issue in Cuetsy repository?

Reply:
I added them in Cuetsy and as-code

Reply:
Thanks so much! :)